pom.xml

```diff
@@ -104,7 +104,7 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-runtime_${scala.binary.version}</artifactId>
+			<artifactId>flink-runtime</artifactId>
 			<version>${flink.version}</version>
 			<type>test-jar</type>
 		</dependency>
```

Comment:
I've tried removing this whole dependency in order to locate benchmarks that have been affected by this bug, and the code was still compiling? Am I missing something?

Reply:
it could actually be that we don't need it. If they compile anyway then we should be able to remove it.

Reply:
The benchmark base is actually in the test-jar.

Reply:
There are two `flink-streaming-java` dependencies. One of default scope.
May it be that there are two `flink-runtime` dependencies which was introduced by `flink-streaming-java` and this direct `flink-runtime` dependency. And conflicts happened?

Reply:
Without this dependency we it as a transitive dependency:
```
[INFO] +- org.apache.flink:flink-test-utils_2.11:jar:1.14-SNAPSHOT:compile
[INFO] |  +- org.apache.flink:flink-runtime:test-jar:tests:1.14-SNAPSHOT:compile
```

Reply:
So we had 2 different flink-runtime test-jar dependencies on the CP, but only one of them could fully work because only 1 version of flink-runtime was there.

Reply:
TL;DR: let's just remove it for now. (although I'm not a huge fan of used undeclared dependencies, precisely because of such surprises...)